### ISSUE TITLE
batch scribt for webdav connection

### ISSUE THREAD

**Issue by mzuske** [meganz/MEGAcmd#437]
hello, i am new with the mega cmd. i am tried to creat a loacal storage (a kind of networkstorage) via webdav in my windows explorer. 

i can enter a folder for webdav via cmd tool. and i createt a storage connection in my explorer. but now i noticed that after a stutdown or reboot, i have to open the cmd tool, login and "webdav" the folder again to use the created storage.

is it possible or is there any kind of "tool" (maybe batchfile) that will do this automatic after the windows start?

thanks for help

**Comment by mattw-mega** [comment]
Yes, the installation in includes `mega-webdav` which you can run from the command line or from a batch/cmd file.  It will resume your old session so long as you didn't explicitly log out (or cancel the session from the webclient).
